tools/SourceKit/tools/sourcekitd-test/sourcekitd-test.cpp

```diff
@@ -1121,6 +1119,11 @@ static int handleTestInvocation(TestOptions Opts, TestOptions &InitOpts) {
                                              Opts.SourceFile.c_str());
   }
 
+  if (!Opts.PrimaryFile.empty()) {
+    sourcekitd_request_dictionary_set_string(Req, KeyPrimaryFile,
+                                             Opts.PrimaryFile.c_str());
```

Comment:
Did we previously set `KeyPrimaryFile` to an empty string if no primary file was passed?

Reply:
We didn't set it at all. `KeyPrimaryFile` is new and I only added it for refactorings (see the removal in this file).